rust-semantics/rust-common-syntax.md

```diff
@@ -540,7 +545,20 @@ https://doc.rust-lang.org/reference/items/extern-crates.html
 
 ```k
 
-  syntax StructExpression ::= "TODO: not needed yet, not implementing"
+  syntax StructExpression ::= StructExprStruct 
+                            | StructExprTuple 
+                            | StructExprUnit
+  syntax StructExprStruct ::= TypePath "{" MaybeStructExprFieldsOrStructBase "}" //TODO: Not implemented properly
+                                                                                 //Should use PathInExpression instead of
+                                                                                 //TypePath.
+  syntax MaybeStructExprFieldsOrStructBase ::= "" | StructExprFields | StructBases
```

Comment:
I think that there is at most one StructBase here. It's fine to keep it as-is, but add a "//TODO: Not implemented properly". I think that you can also have `StructExprFields , StructBase` and `StructExprFields , ` here. Again, we don't need them, but you should add the TODO. Also, StructBase below is actually ".. Expression". You should probably not change it, but you should add a TODO.